.gitignore

```diff
@@ -20,8 +20,8 @@ doc/
 !ngraph/doc
 docs/build_documentation/work_dir/
 inference-engine/plugins/
-inference-engine/temp
 inference-engine/report
+temp/
```

Comment:
is it required? Looks like `temp` should be presented in `${BUILD_DIR}/temp` only, not in repo root

Reply:
Yes I would like to be able to use OV_TEMP option to change temp directory. Because for the developer point build folder is not the best place, because you can remove it to regenerate cmake project.